web/css/main.css

```diff
@@ -10,7 +10,7 @@
 @import url("../../node_modules/jquery.joyride/jquery.joyride.css");
 @import url("../../node_modules/jquery-jcrop/css/jquery.Jcrop.css");
 @import url("../../node_modules/ol/ol.css");
-@import url("../../node_modules/nouislider-browser/jquery.nouislider.css");
+@import url("../../node_modules/nouislider/distribute/jquery.nouislider.min.css");
```

Comment:
If there's a non-minified version of CSS, you should prefer that, since the build script minifies and generates a source map that points to the original file. If we include the minified version, we lose the ability to use the source map.